examples/random_pick/random_pick/launch/demo/gpd.launch

```diff
@@ -0,0 +1,77 @@
+<launch>
+
+  <arg name="device"                     default="0"
+        doc="target device to execute inference. [0:CPU]|1:GPU|2:VPU|3:FPGA" />
+  <arg name="plane_remove"               default="false"
+        doc="if remove plane from PCL inputs" />
+
+  <!-- Load hand geometry parameters -->  
+  <include file="$(find random_pick)/launch/demo/hand_geometry.launch">
+    <arg name="node" value="detect_grasps" />
+  </include>
+
+  <!-- Load classifier parameters -->
+  <include file="$(find gpd)/launch/caffe/classifier_15channels.launch">
+    <arg name="node" value="detect_grasps" />
+  </include>
+  
+	<node name="detect_grasps" pkg="gpd" type="detect_grasps" output="screen">
+
+    <param name="device" type="int" value="$(arg device)" />
+    <param name="plane_remove" type="bool" value="$(arg plane_remove)" />
+
+    <!-- If sequential importance sampling is used (default: false) -->
+    <param name="use_importance_sampling" value="false" />
+        
+    <!-- What type of point cloud is used and what ROS topic it comes from -->
+    <param name="cloud_type" value="0" /> <!-- 0: PointCloud2, 1: CloudSized, 2: CloudIndexed, 3: CloudSamples -->
+    <param name="cloud_topic" value="/camera/depth_registered/points" />
+    
+    <!-- (optional) The ROS topic that the samples come from (default: an empty string) -->
+    <param name="samples_topic" value="" />
+    
+    <!-- Plotting parameters -->
+    <param name="plot_normals" value="false" />
```

Comment:
Nit: These would be good candidates to put in a yaml

Reply:
Great idea! The parameters have been moved to `gpd.yaml` file.